recipes/mlpack/meta.yaml

```diff
@@ -20,20 +20,20 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake
   host:
-    - cython
+    - cython >=0.24
     - pip
     - boost
     - python
     - numpy
-    - pandas  > 0.15.0
-    - armadillo
+    - pandas  >=0.15.0
+    - armadillo >=6.500.0
     - pytest-runner
   run:
     - boost
     - python
     - numpy
-    - pandas
-    - armadillo
+    - pandas  >=0.15.0
+    - armadillo >=6.500.0
```

Comment:
This should be `{{ pin_compatible('armadillo') }}`. Can be done in the feedstock.

Reply:
got it! Thanks